src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -313,20 +308,13 @@ const UniqueTokenExpandedState = ({
       {ios && (
         <BlurWrapper height={deviceHeight} width={deviceWidth}>
           <BackgroundImage>
-            {isSVG ? (
```

Comment:
this was not needed as we have this logic already in the `UniqueTokenImage` component, you can verify by looking at svg NFTs like Blitmap and Shields